lib/target/users.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Target.Users do
+  @moduledoc """
+  The User context.
+  """
+
+  import Ecto.Query, warn: false
+
+  alias Target.Repo
+  alias Target.Users.User
+
+  def list_users do
+    Repo.all(User)
+  end
+
+  def get_by_email(email) do
+    Repo.get_by(User, email: email)
+  end
+
+  def current_email_unconfirmed?(%{
```

Comment:
I think it would be better to use the positive version `current_email_confirmed?`, negative predicates make logic complicated